### ISSUE TITLE
Phalcon Devtool 3.0.0 (3.0.x-dev) Migration Run error

### ISSUE THREAD

**Issue by idetomohiro** [phalcon/phalcon-devtools#727]
When I executed migration command using Phalcon Devtools 3.0.0(3.0.x-dev), an error occurred.

command:
```
$ phalcon migration run

error:
PHP Fatal error: Allowed memory size of 1073741824 bytes exhausted (tried to allocate 14560458 bytes) in /var/www/vendor/phalcon/devtools/scripts/Phalcon/Version/IncrementalItem.php on line 63
```
I think the cause is the line 76 of ItemCollection.php
https://github.com/phalcon/phalcon-devtools/blob/master/scripts/Phalcon/Version/ItemCollection.php#L76

In the constructor of IncrementalItem, an array is given for $options, not an integer.
So, in the line 61 (https://github.com/phalcon/phalcon-devtools/blob/master/scripts/Phalcon/Version/IncrementalItem.php#L61)
the for statement loops infinitely, and the variable $version consumes entire memory.

**Comment by sergeyklay** [comment]
I'll try to sort out as soon as I can

**Comment by sergeyklay** [comment]
Fixed in the `3.0.x` branch.
